nixos/modules/services/audio/mpd.nix

```diff
@@ -11,6 +11,8 @@ let
   cfg = config.services.mpd;
 
   mpdConf = pkgs.writeText "mpd.conf" ''
+    # This file was automatically generated by nixos. Edit mpd's configuration via NixOS' configuration.nix
```

Comment:
Just a small nitpick: You should use a consistent spelling of NixOS

Reply:
I agree :).